app/views/reports/_census_period.html.erb

```diff
@@ -1,13 +1,16 @@
+<h2 class="govuk-heading-m">
+  Census Sign Off
+</h2>
 <ul class="govuk-list govuk-list--spaced">
   <li>
     <p>
       <%= govuk_link_to "New trainees for the #{@current_academic_cycle_label} academic year", itt_new_starter_data_sign_off_reports_path, class: "new-trainee-data" %>
-      - for census sign off
+      - for performance profiles sign off.
```

Comment:
The copy doesn't seem to match the title and filename. Should this say performance profiles or census?

Reply:
Fixed we need to review census when we get around to it